install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,410 @@
 ////
 Module included in the following assemblies:
-
 install_config/configuring_rhv.adoc
 ////
-
 [id='configuring-rhv-objects_{context}']
-= Configuring Red Hat Virtualization objects
+= Configuring Red Hat Virtualization
+
+To integrate {product-title} with Red Hat Virtualization, perform the following actions as part of your xref:../install/host_preparation.adoc#install-config-install-host-preparation[host preparation].
+
+== Creating the bastion virtual machine
+
+The bastion virtual machine is used to install {product-title} on Red Hat Virtualization.
+
+.Procedure
+
+. Log in to the Manager machine using SSH.
+. Create a temporary bastion installation directory, for example, *_/bastion_installation_*, for the installation files.
+. Create an encrypted file, *_/bastion_installation/secure_vars.yaml_*:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ansible-vault create secure_vars.yaml
+----
+
+. Add the following variables to *_secure_vars.yaml_*:
++
+[source,yml]
+----
+engine_password: <Manager_password>
+bastion_root_password: <bastion_root_password>
+rhsub_user: <Red_Hat_Subscription_Manager_username>
+rhsub_pass: <Red_Hat_Subscription_Manager_password>
+rhsub_pool: <Red_Hat_Subscription_Manager_pool_id>
+root_password: <OpenShift_node_root_password>
+----
+
+. Save the file and record the vault password.
+
+. Create the installation playbook, *_/bastion_installation/create-bastion-machine-playbook.yaml_*, and update its parameters:
++
+[source,yml]
+----
+---
+- name: Create a bastion machine
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    engine_url: https://_Manager_FQDN_/ovirt-engine/api
+    engine_user: <admin@internal>
+    engine_password: "{{ engine_password }}"
+    engine_cafile: /etc/pki/ovirt-engine/ca.pem
+
+    qcow_url: <1>
+    template_cluster: Default
+    template_name: rhelguest7
+    template_memory: 4GiB
+    template_cpu: 2
+    wait_for_ip: true
+    debug_vm_create: false
 
-To integrate {product-title} with Red Hat Virtualization, take the following
-actions as part of your
-xref:../install/host_preparation.adoc#install-config-install-host-preparation[host preparation].
+    vms:
+    - name: rhel-bastion
+      cluster: Default
```

Comment:
@rgolangh I believe we should just reference `"{{ template_cluster }}"` here in order not to duplicate the information.

Reply:
+1

Reply:
done